backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/KonanDriver.kt

```diff
@@ -80,10 +105,19 @@ fun runTopLevelPhases(konanConfig: KonanConfig, environment: KotlinCoreEnvironme
     phaser.phase(KonanPhase.GEN_SYNTHETIC_FIELDS) {
         markBackingFields(context)
     }
+    phaser.phase(KonanPhase.COPY_DEFAULT_VALUES_TO_ACTUAL) {
```

Comment:
Why do we need it this early?

Reply:
I have placed the following comment there:
    // TODO: We copy default value expressions from expects to actuals before IR serialization,
    // because the current infrastructure doesn't allow us to get them at deserialization stage.
    // That requires some design and implementation work.